### PR TITLE
install black and flake8

### PR DESCRIPTION
**Pull request by JackDuluoz**
Set up black and flake8 config files.

### CODE REVIEW

**Review by jekiquin — APPROVED** [review]
LGTM! Great work Michael!

File: README.md [hunk 1]
```diff
@@ -1 +1,25 @@
-# backend-template
+Follow these steps:
+
+1. You just created a new repo using this template.
```
Comment (chunloy) [inline README.md:3]: Maybe add a command to clone this template? e.g. `git clone <template>`

File: README.md
Comment (chunloy) [inline README.md:1]: Include some placeholder sections similar to frontend `README`

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/venv
```
Comment (chunloy) [inline .gitignore:1]: Not too familiar with what goes in here for python projects, but definitely add: 
- .DS_Store 
- \_\_pycache\_\_
- .serverless
- node_modules

**Comment by nicolepa** [comment]
testing review+approval process...

**Comment by nicolepa** [comment]
testing GH review+approve process...

**Comment by nicolepa** [comment]
mistake; rollback

**Review by nicolepa — APPROVED** [review]

**Pull request merged**